CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- [cygnus-common][MySQLBackend] Implement connection pooling (#1600)
```

Comment:
Note that 1600 is a self-reference to the PR itself. Actually, I think you should include references to the issues this PR is fixing, I mean #1598 and #1597 (onel ine per issue).

Reply:
Done f014b0b